src/Apps/Artist/Components/__test__/NavigationTabs.test.tsx

```diff
@@ -1,10 +1,10 @@
+import { NavigationTabsFixture } from "Apps/__test__/Fixtures/Artist/Components/NavigationTabs"
+import { NavigationTabsFragmentContainer as NavigationTabs } from "Apps/Artist/Components/NavigationTabs"
+import { renderRelayTree } from "DevTools"
 import { graphql } from "react-relay"
-import { renderRelayTree } from "../../../../DevTools"
```

Comment:
Thanks for these, I was actually getting an issue with VSCode complaining it couldn't find the files when specified absolutely, but I should have looked into it more. My editor was probably in a bad state, this works perfectly now.